src/controllers/blocks/BlocksExtrinsicsController.ts

```diff
@@ -1,13 +1,27 @@
 import { ApiPromise } from '@polkadot/api';
 import { RequestHandler } from 'express';
+import { BlockWeightStore } from 'src/types/chains-config';
 
 import { BlocksService } from '../../services';
 import { INumberParam } from '../../types/requests';
 import AbstractController from '../AbstractController';
 
+interface ControllerOptions {
+	blockWeightStore: BlockWeightStore;
+	minCalcFeeRuntime: null | number;
```

Comment:
What is our convention for these cases, isn't there `Option<Number>` in TS? We don't like it?

Reply:
Yea we can use it here, and I updated the code to use it here. 

`export type Option<T> = T | null;` 

And then set it to be Option<number>

Reply:
@dvdplm If you have seen an option in the code before it is from polkadot-js which has an `Option` class which extends `Codec` - we mainly see it as a return type from storage queries.

@TarikGul if you add `Option` just make sure there is not a name space collision with the polkadot-js ones. Maybe we can do `SASOption`, 'IOption` etc.